package.json

```diff
@@ -49,7 +49,9 @@
   },
   "resolutions": {
     "babel-core": "7.0.0-bridge.0",
+    "domelementtype": "1.3.1",
     "graphql": "0.13.2",
+    "htmlparser2": "3.10.1",
```

Comment:
Is this `htmlparser2` and the `domelementtype` above used, or maybe I missed them somewhere?

Reply:
Ah! These resolutions are to reduce the number of duplicates in the bundle, since they're required by a few modules. 😄